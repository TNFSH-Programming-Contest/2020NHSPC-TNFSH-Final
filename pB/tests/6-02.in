880
-538.84 304.297
-540.999 300.442
-543.13 296.571
-545.234 292.686
-547.31 288.786
-549.358 284.87
-551.377 280.941
-553.369 276.997
-555.333 273.039
-557.268 269.067
-559.175 265.081
-561.054 261.082
-562.903 257.069
-564.724 253.044
-566.517 249.005
-568.28 244.954
-570.015 240.89
-571.72 236.814
-573.396 232.726
-575.043 228.626
-576.661 224.515
-578.249 220.392
-579.808 216.257
-581.338 212.112
-582.837 207.956
-584.307 203.789
-585.747 199.612
-587.158 195.425
-588.538 191.228
-589.888 187.021
-591.209 182.804
-592.499 178.578
-593.759 174.343
-594.988 170.099
-596.188 165.847
-597.357 161.586
-598.495 157.317
-599.603 153.04
-600.68 148.755
-601.727 144.462
-602.743 140.162
-603.729 135.855
-604.683 131.541
-605.607 127.22
-606.5 122.893
-607.362 118.559
-608.193 114.22
-608.993 109.874
-609.762 105.523
-610.5 101.167
-611.207 96.806
-611.882 92.439
-612.527 88.068
-613.14 83.692
-613.722 79.312
-614.272 74.929
-614.792 70.541
-615.28 66.149
-615.736 61.755
-616.162 57.357
-616.555 52.956
-616.918 48.552
-617.249 44.146
-617.548 39.738
-617.816 35.328
-618.053 30.916
-618.258 26.502
-618.431 22.087
-618.573 17.671
-618.683 13.254
-618.762 8.837
-618.81 4.418
-618.825 0.0
-618.81 -4.418
-618.762 -8.837
-618.683 -13.254
-618.573 -17.671
-618.431 -22.087
-618.258 -26.502
-618.053 -30.916
-617.816 -35.328
-617.548 -39.738
-617.249 -44.146
-616.918 -48.552
-616.555 -52.956
-616.162 -57.357
-615.736 -61.755
-615.28 -66.149
-614.792 -70.541
-614.272 -74.929
-613.722 -79.312
-613.14 -83.692
-612.527 -88.068
-611.882 -92.439
-611.207 -96.806
-610.5 -101.167
-609.762 -105.523
-608.993 -109.874
-608.193 -114.22
-607.362 -118.559
-606.5 -122.893
-605.607 -127.22
-604.683 -131.541
-603.729 -135.855
-602.743 -140.162
-601.727 -144.462
-600.68 -148.755
-599.603 -153.04
-598.495 -157.317
-597.357 -161.586
-596.188 -165.847
-594.988 -170.099
-593.759 -174.343
-592.499 -178.578
-591.209 -182.804
-589.888 -187.021
-588.538 -191.228
-587.158 -195.425
-585.747 -199.612
-584.307 -203.789
-582.837 -207.956
-581.338 -212.112
-579.808 -216.257
-578.249 -220.392
-576.661 -224.515
-575.043 -228.626
-573.396 -232.726
-571.72 -236.814
-570.015 -240.89
-568.28 -244.954
-566.517 -249.005
-564.724 -253.044
-562.903 -257.069
-561.054 -261.082
-559.175 -265.081
-557.268 -269.067
-555.333 -273.039
-553.369 -276.997
-551.377 -280.941
-549.358 -284.87
-547.31 -288.786
-545.234 -292.686
-543.13 -296.571
-540.999 -300.442
-538.84 -304.297
-536.653 -308.136
-534.44 -311.96
-532.199 -315.768
-529.931 -319.56
-527.635 -323.335
-525.313 -327.094
-522.965 -330.837
-520.589 -334.562
-518.187 -338.271
-515.759 -341.962
-513.304 -345.636
-510.823 -349.292
-508.316 -352.93
-505.783 -356.55
-503.225 -360.153
-500.64 -363.736
-498.03 -367.302
-495.395 -370.848
-492.735 -374.376
-490.049 -377.884
-487.339 -381.374
-484.603 -384.844
-481.843 -388.294
-479.058 -391.724
-476.249 -395.135
-473.416 -398.525
-470.559 -401.895
-467.677 -405.244
-464.772 -408.573
-461.843 -411.881
-458.89 -415.168
-455.914 -418.434
-452.915 -421.679
-449.893 -424.902
-446.847 -428.103
-443.779 -431.283
-440.689 -434.44
-437.576 -437.576
-434.44 -440.689
-431.283 -443.779
-428.103 -446.847
-424.902 -449.893
-421.679 -452.915
-418.434 -455.914
-415.168 -458.89
-411.881 -461.843
-408.573 -464.772
-405.244 -467.677
-401.895 -470.559
-398.525 -473.416
-395.135 -476.249
-391.724 -479.058
-388.294 -481.843
-384.844 -484.603
-381.374 -487.339
-377.884 -490.049
-374.376 -492.735
-370.848 -495.395
-367.302 -498.03
-363.736 -500.64
-360.153 -503.225
-356.55 -505.783
-352.93 -508.316
-349.292 -510.823
-345.636 -513.304
-341.962 -515.759
-338.271 -518.187
-334.562 -520.589
-330.837 -522.965
-327.094 -525.313
-323.335 -527.635
-319.56 -529.931
-315.768 -532.199
-311.96 -534.44
-308.136 -536.653
-304.297 -538.84
-300.442 -540.999
-296.571 -543.13
-292.686 -545.234
-288.786 -547.31
-284.87 -549.358
-280.941 -551.377
-276.997 -553.369
-273.039 -555.333
-269.067 -557.268
-265.081 -559.175
-261.082 -561.054
-257.069 -562.903
-253.044 -564.724
-249.005 -566.517
-244.954 -568.28
-240.89 -570.015
-236.814 -571.72
-232.726 -573.396
-228.626 -575.043
-224.515 -576.661
-220.392 -578.249
-216.257 -579.808
-212.112 -581.338
-207.956 -582.837
-203.789 -584.307
-199.612 -585.747
-195.425 -587.158
-191.228 -588.538
-187.021 -589.888
-182.804 -591.209
-178.578 -592.499
-174.343 -593.759
-170.099 -594.988
-165.847 -596.188
-161.586 -597.357
-157.317 -598.495
-153.04 -599.603
-148.755 -600.68
-144.462 -601.727
-140.162 -602.743
-135.855 -603.729
-131.541 -604.683
-127.22 -605.607
-122.893 -606.5
-118.559 -607.362
-114.22 -608.193
-109.874 -608.993
-105.523 -609.762
-101.167 -610.5
-96.806 -611.207
-92.439 -611.882
-88.068 -612.527
-83.692 -613.14
-79.312 -613.722
-74.929 -614.272
-70.541 -614.792
-66.149 -615.28
-61.755 -615.736
-57.357 -616.162
-52.956 -616.555
-48.552 -616.918
-44.146 -617.249
-39.738 -617.548
-35.328 -617.816
-30.916 -618.053
-26.502 -618.258
-22.087 -618.431
-17.671 -618.573
-13.254 -618.683
-8.837 -618.762
-4.418 -618.81
-0.0 -618.825
4.418 -618.81
8.837 -618.762
13.254 -618.683
17.671 -618.573
22.087 -618.431
26.502 -618.258
30.916 -618.053
35.328 -617.816
39.738 -617.548
44.146 -617.249
48.552 -616.918
52.956 -616.555
57.357 -616.162
61.755 -615.736
66.149 -615.28
70.541 -614.792
74.929 -614.272
79.312 -613.722
83.692 -613.14
88.068 -612.527
92.439 -611.882
96.806 -611.207
101.167 -610.5
105.523 -609.762
109.874 -608.993
114.22 -608.193
118.559 -607.362
122.893 -606.5
127.22 -605.607
131.541 -604.683
135.855 -603.729
140.162 -602.743
144.462 -601.727
148.755 -600.68
153.04 -599.603
157.317 -598.495
161.586 -597.357
165.847 -596.188
170.099 -594.988
174.343 -593.759
178.578 -592.499
182.804 -591.209
187.021 -589.888
191.228 -588.538
195.425 -587.158
199.612 -585.747
203.789 -584.307
207.956 -582.837
212.112 -581.338
216.257 -579.808
220.392 -578.249
224.515 -576.661
228.626 -575.043
232.726 -573.396
236.814 -571.72
240.89 -570.015
244.954 -568.28
249.005 -566.517
253.044 -564.724
257.069 -562.903
261.082 -561.054
265.081 -559.175
269.067 -557.268
273.039 -555.333
276.997 -553.369
280.941 -551.377
284.87 -549.358
288.786 -547.31
292.686 -545.234
296.571 -543.13
300.442 -540.999
304.297 -538.84
308.136 -536.653
311.96 -534.44
315.768 -532.199
319.56 -529.931
323.335 -527.635
327.094 -525.313
330.837 -522.965
334.562 -520.589
338.271 -518.187
341.962 -515.759
345.636 -513.304
349.292 -510.823
352.93 -508.316
356.55 -505.783
360.153 -503.225
363.736 -500.64
367.302 -498.03
370.848 -495.395
374.376 -492.735
377.884 -490.049
381.374 -487.339
384.844 -484.603
388.294 -481.843
391.724 -479.058
395.135 -476.249
398.525 -473.416
401.895 -470.559
405.244 -467.677
408.573 -464.772
411.881 -461.843
415.168 -458.89
418.434 -455.914
421.679 -452.915
424.902 -449.893
428.103 -446.847
431.283 -443.779
434.44 -440.689
437.576 -437.576
440.689 -434.44
443.779 -431.283
446.847 -428.103
449.893 -424.902
452.915 -421.679
455.914 -418.434
458.89 -415.168
461.843 -411.881
464.772 -408.573
467.677 -405.244
470.559 -401.895
473.416 -398.525
476.249 -395.135
479.058 -391.724
481.843 -388.294
484.603 -384.844
487.339 -381.374
490.049 -377.884
492.735 -374.376
495.395 -370.848
498.03 -367.302
500.64 -363.736
503.225 -360.153
505.783 -356.55
508.316 -352.93
510.823 -349.292
513.304 -345.636
515.759 -341.962
518.187 -338.271
520.589 -334.562
522.965 -330.837
525.313 -327.094
527.635 -323.335
529.931 -319.56
532.199 -315.768
534.44 -311.96
536.653 -308.136
538.84 -304.297
540.999 -300.442
543.13 -296.571
545.234 -292.686
547.31 -288.786
549.358 -284.87
551.377 -280.941
553.369 -276.997
555.333 -273.039
557.268 -269.067
559.175 -265.081
561.054 -261.082
562.903 -257.069
564.724 -253.044
566.517 -249.005
568.28 -244.954
570.015 -240.89
571.72 -236.814
573.396 -232.726
575.043 -228.626
576.661 -224.515
578.249 -220.392
579.808 -216.257
581.338 -212.112
582.837 -207.956
584.307 -203.789
585.747 -199.612
587.158 -195.425
588.538 -191.228
589.888 -187.021
591.209 -182.804
592.499 -178.578
593.759 -174.343
594.988 -170.099
596.188 -165.847
597.357 -161.586
598.495 -157.317
599.603 -153.04
600.68 -148.755
601.727 -144.462
602.743 -140.162
603.729 -135.855
604.683 -131.541
605.607 -127.22
606.5 -122.893
607.362 -118.559
608.193 -114.22
608.993 -109.874
609.762 -105.523
610.5 -101.167
611.207 -96.806
611.882 -92.439
612.527 -88.068
613.14 -83.692
613.722 -79.312
614.272 -74.929
614.792 -70.541
615.28 -66.149
615.736 -61.755
616.162 -57.357
616.555 -52.956
616.918 -48.552
617.249 -44.146
617.548 -39.738
617.816 -35.328
618.053 -30.916
618.258 -26.502
618.431 -22.087
618.573 -17.671
618.683 -13.254
618.762 -8.837
618.81 -4.418
618.825 0.0
618.81 4.418
618.762 8.837
618.683 13.254
618.573 17.671
618.431 22.087
618.258 26.502
618.053 30.916
617.816 35.328
617.548 39.738
617.249 44.146
616.918 48.552
616.555 52.956
616.162 57.357
615.736 61.755
615.28 66.149
614.792 70.541
614.272 74.929
613.722 79.312
613.14 83.692
612.527 88.068
611.882 92.439
611.207 96.806
610.5 101.167
609.762 105.523
608.993 109.874
608.193 114.22
607.362 118.559
606.5 122.893
605.607 127.22
604.683 131.541
603.729 135.855
602.743 140.162
601.727 144.462
600.68 148.755
599.603 153.04
598.495 157.317
597.357 161.586
596.188 165.847
594.988 170.099
593.759 174.343
592.499 178.578
591.209 182.804
589.888 187.021
588.538 191.228
587.158 195.425
585.747 199.612
584.307 203.789
582.837 207.956
581.338 212.112
579.808 216.257
578.249 220.392
576.661 224.515
575.043 228.626
573.396 232.726
571.72 236.814
570.015 240.89
568.28 244.954
566.517 249.005
564.724 253.044
562.903 257.069
561.054 261.082
559.175 265.081
557.268 269.067
555.333 273.039
553.369 276.997
551.377 280.941
549.358 284.87
547.31 288.786
545.234 292.686
543.13 296.571
540.999 300.442
538.84 304.297
536.653 308.136
534.44 311.96
532.199 315.768
529.931 319.56
527.635 323.335
525.313 327.094
522.965 330.837
520.589 334.562
518.187 338.271
515.759 341.962
513.304 345.636
510.823 349.292
508.316 352.93
505.783 356.55
503.225 360.153
500.64 363.736
498.03 367.302
495.395 370.848
492.735 374.376
490.049 377.884
487.339 381.374
484.603 384.844
481.843 388.294
479.058 391.724
476.249 395.135
473.416 398.525
470.559 401.895
467.677 405.244
464.772 408.573
461.843 411.881
458.89 415.168
455.914 418.434
452.915 421.679
449.893 424.902
446.847 428.103
443.779 431.283
440.689 434.44
437.576 437.576
434.44 440.689
431.283 443.779
428.103 446.847
424.902 449.893
421.679 452.915
418.434 455.914
415.168 458.89
411.881 461.843
408.573 464.772
405.244 467.677
401.895 470.559
398.525 473.416
395.135 476.249
391.724 479.058
388.294 481.843
384.844 484.603
381.374 487.339
377.884 490.049
374.376 492.735
370.848 495.395
367.302 498.03
363.736 500.64
360.153 503.225
356.55 505.783
352.93 508.316
349.292 510.823
345.636 513.304
341.962 515.759
338.271 518.187
334.562 520.589
330.837 522.965
327.094 525.313
323.335 527.635
319.56 529.931
315.768 532.199
311.96 534.44
308.136 536.653
304.297 538.84
300.442 540.999
296.571 543.13
292.686 545.234
288.786 547.31
284.87 549.358
280.941 551.377
276.997 553.369
273.039 555.333
269.067 557.268
265.081 559.175
261.082 561.054
257.069 562.903
253.044 564.724
249.005 566.517
244.954 568.28
240.89 570.015
236.814 571.72
232.726 573.396
228.626 575.043
224.515 576.661
220.392 578.249
216.257 579.808
212.112 581.338
207.956 582.837
203.789 584.307
199.612 585.747
195.425 587.158
191.228 588.538
187.021 589.888
182.804 591.209
178.578 592.499
174.343 593.759
170.099 594.988
165.847 596.188
161.586 597.357
157.317 598.495
153.04 599.603
148.755 600.68
144.462 601.727
140.162 602.743
135.855 603.729
131.541 604.683
127.22 605.607
122.893 606.5
118.559 607.362
114.22 608.193
109.874 608.993
105.523 609.762
101.167 610.5
96.806 611.207
92.439 611.882
88.068 612.527
83.692 613.14
79.312 613.722
74.929 614.272
70.541 614.792
66.149 615.28
61.755 615.736
57.357 616.162
52.956 616.555
48.552 616.918
44.146 617.249
39.738 617.548
35.328 617.816
30.916 618.053
26.502 618.258
22.087 618.431
17.671 618.573
13.254 618.683
8.837 618.762
4.418 618.81
0.0 618.825
-4.418 618.81
-8.837 618.762
-13.254 618.683
-17.671 618.573
-22.087 618.431
-26.502 618.258
-30.916 618.053
-35.328 617.816
-39.738 617.548
-44.146 617.249
-48.552 616.918
-52.956 616.555
-57.357 616.162
-61.755 615.736
-66.149 615.28
-70.541 614.792
-74.929 614.272
-79.312 613.722
-83.692 613.14
-88.068 612.527
-92.439 611.882
-96.806 611.207
-101.167 610.5
-105.523 609.762
-109.874 608.993
-114.22 608.193
-118.559 607.362
-122.893 606.5
-127.22 605.607
-131.541 604.683
-135.855 603.729
-140.162 602.743
-144.462 601.727
-148.755 600.68
-153.04 599.603
-157.317 598.495
-161.586 597.357
-165.847 596.188
-170.099 594.988
-174.343 593.759
-178.578 592.499
-182.804 591.209
-187.021 589.888
-191.228 588.538
-195.425 587.158
-199.612 585.747
-203.789 584.307
-207.956 582.837
-212.112 581.338
-216.257 579.808
-220.392 578.249
-224.515 576.661
-228.626 575.043
-232.726 573.396
-236.814 571.72
-240.89 570.015
-244.954 568.28
-249.005 566.517
-253.044 564.724
-257.069 562.903
-261.082 561.054
-265.081 559.175
-269.067 557.268
-273.039 555.333
-276.997 553.369
-280.941 551.377
-284.87 549.358
-288.786 547.31
-292.686 545.234
-296.571 543.13
-300.442 540.999
-304.297 538.84
-308.136 536.653
-311.96 534.44
-315.768 532.199
-319.56 529.931
-323.335 527.635
-327.094 525.313
-330.837 522.965
-334.562 520.589
-338.271 518.187
-341.962 515.759
-345.636 513.304
-349.292 510.823
-352.93 508.316
-356.55 505.783
-360.153 503.225
-363.736 500.64
-367.302 498.03
-370.848 495.395
-374.376 492.735
-377.884 490.049
-381.374 487.339
-384.844 484.603
-388.294 481.843
-391.724 479.058
-395.135 476.249
-398.525 473.416
-401.895 470.559
-405.244 467.677
-408.573 464.772
-411.881 461.843
-415.168 458.89
-418.434 455.914
-421.679 452.915
-424.902 449.893
-428.103 446.847
-431.283 443.779
-434.44 440.689
-437.576 437.576
-440.689 434.44
-443.779 431.283
-446.847 428.103
-449.893 424.902
-452.915 421.679
-455.914 418.434
-458.89 415.168
-461.843 411.881
-464.772 408.573
-467.677 405.244
-470.559 401.895
-473.416 398.525
-476.249 395.135
-479.058 391.724
-481.843 388.294
-484.603 384.844
-487.339 381.374
-490.049 377.884
-492.735 374.376
-495.395 370.848
-498.03 367.302
-500.64 363.736
-503.225 360.153
-505.783 356.55
-508.316 352.93
-510.823 349.292
-513.304 345.636
-515.759 341.962
-518.187 338.271
-520.589 334.562
-522.965 330.837
-525.313 327.094
-527.635 323.335
-529.931 319.56
-532.199 315.768
-534.44 311.96
-536.653 308.136
-86.397 302.205
