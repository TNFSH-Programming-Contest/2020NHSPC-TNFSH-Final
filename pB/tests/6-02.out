2735527.595